Azure/Azure-AppServices-Diagnostics-Portal
# Curibe/opt insights integration diag solve

## Overview

Removing empty banner being displayed to non-beta subscriptions.

Non-beta subscriptions are seeing an empty banner
## Related Work Item

N/A
## Type of change




## Solution description

Added the check for beta subscriptions to app-insights-enablement component

## How Can This Be Tested? <span>&#128269;</span>



1. In the Azure Portal, under a non-beta subscription, go to a Web App integrated with Application Insights which has profilers enabled and Code Optimization Insights.
2. Click on Diagnose and Solve problems then in Availability and Performance. This will load the Overview detector.
3.  In the Overview detector first it will try to show Application Insights there should not be an empty banner under the Application Insights banner.

## Checklist <span>&#9989;</span>

## Screenshots Before Fix (if appropriate):
![image](https://user-images.githubusercontent.com/7376302/235778041-f476e197-655c-4bd7-86f2-3632a83f1699.png)

## Screenshots After Fix (if appropriate):

## Extra Notes
